docs/modules/usage/configuration-options.md

```diff
@@ -178,8 +177,11 @@ The core configuration options are defined in the `[core]` section of the `confi
   - Description: JWT secret for authentication. Please set it to your own value.
 
 ## LLM Configuration
+
 The LLM (Large Language Model) configuration options are defined in the `[llm]` section of the `config.toml` file.
 
+To use these with the docker command, pass in `-e LLM_<option>`. Example: `-e LLM_API_KEY`.
+
```

Comment:
While this is accurate, we may want to choose an example that isn't available in the UI Settings. Like LLM_MAX_RETRIES maybe?

Reply:
Updated per your suggestion. LLM_NUM_RETRIES. Thanks!